scripts/odemis-start-turret.py

```diff
@@ -64,16 +66,41 @@
 
 def main(args):
     turret = int(args[1])
+    notify = "--notify" in args[1:]
+    if len(args) >= 4:
+        name = args[3]
+    else:
+        name = f"turret {turret}"
     try:
+        if notify:
+            notify2.init("Odemis")
```

Comment:
Can this line be after line 79 i.e. turning on power control unit such that it can be unified with second if notify on line 80?